aslam_cv_frames/include/aslam/frames/visual-frame.h

```diff
@@ -99,9 +100,19 @@ class VisualFrame  {
   /// The keypoint measurements stored in a frame.
   const Eigen::Matrix2Xd& getKeypointMeasurements() const;
 
-  /// Get the number of keypoints measurements stored in this frame.
+  /// Get the number of keypoint measurements stored in this frame.
   inline size_t getNumKeypointMeasurements() const {
-    return static_cast<size_t>(getKeypointMeasurements().cols());
+    return hasKeypointMeasurements() ? getKeypointMeasurements().cols() : 0u;
```

Comment:
WDYT @smauq. This `hasKeypointMeasurements()` seems to be still a good thing. Do you agree? I also added now a `getTotalNumKeypointMeasurements` for convenience.

Reply:
Yeah, looks good!